digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1022,6 +1022,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
             ProjectDAO.getProjectLinksByLinkId(updatedProjectLinks.head.linkId).headOption match {
               case Some(roadPartLink) =>
                 if (roadPartLink.roadNumber == newRoadNumber && roadPartLink.roadPartNumber == newRoadPartNumber)
+                  if (ProjectDAO.getProjectLinksByProjectAndLinkId(linkIds,projectId).exists(x=>x.status!=LinkStatus.Numbering)) //check if renumbering has already been done (and we are actually doing ie direction change)
```

Comment:
Should we instead check that the renumbering is not trying to use the same number as the referenced road address object?

Reply:
that would be cleaner way

Reply:
i will change the logic to that